app/controllers/builder_controller.rb

```diff
@@ -75,7 +75,12 @@ def create_charge
       amount: Postcard::PRICE,
       currency: 'usd',
       description: STRIPE_CHARGE_DESCRIPTION,
-      source: token
+      source: token,
+      metadata: {
+        postcard_id: @postcard&.id,
```

Comment:
I don't think the safe navigation operator should be required in production. We should always have a valid `Postcard` and authenticated `User`. But Postcards aren't created yet in the `Builder` flow so for now this crashes without the `&`.

`email` and `postcard_id` are null so they don't show up yet in stripe, but they do when I hardcode values:
<img width="456" alt="Screen Shot 2019-11-13 at 7 13 47 PM" src="https://user-images.githubusercontent.com/950405/68823820-d2f6f280-0649-11ea-9ae9-383dbc1f9024.png">

Reply:
Actually...

I think this is just a bug on my end. I should be using `postcard` instead of `@postcard` here and it should be set. I'll try that in a sec